DataFormats/GEMDigi/interface/AMCdata.h

```diff
@@ -92,7 +92,7 @@ namespace gem {
     uint8_t amcNum() const { return AMCheader1{amch1_}.AMCnum; }
 
     uint16_t boardId() const { return AMCheader2{amch2_}.boardID; }
-    uint16_t orbitNum() const { return AMCheader2{amch2_}.orbitNum; }
+    int16_t orbitNum() const { return AMCheader2{amch2_}.orbitNum; }
```

Comment:
everything else is uint\*; is orbitNum special?

Reply:
not special, just the default MC value is -1

Reply:
but so is `bunchCrossing()`; to be consistent, shouldn't the bx data member be changed as well?

Reply:
I reverted back for now. 
I changed orbitNum because it was 16 bits while bunchCrossing is only 12 bits.